advanced_functionality/pytorch_deploy_large_GPT_model/GPT-J-6B-model-parallel-inference-DJL.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p><code>The creation of the SageMaker endpoint might take a while</code> - is there a time estimate for how long it will take to deploy the endpoint? 1 minute, 5 minutes, 30 minutes, etc? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
because we use large GPU instances, time is almost impossible to estimate. Customer might runs into ICE issue and have to re-run.